pinot-server/src/main/java/org/apache/pinot/server/api/resources/TablesResource.java

```diff
@@ -175,4 +183,41 @@ public String getCrcMetadataForTable(
       }
     }
   }
+
+  @GET
+  @Produces(MediaType.APPLICATION_OCTET_STREAM)
+  @Path("/tables/{tableName}/segments/{segmentName}")
```

Comment:
this path is the deprecated one in the controller. Can we use the same as what the controller uses?
`/segments/{tableName}/{segmentName}`. Please add this as a discussion point in your doc, and start an email discussion on this spec. Within a couple of days, we can get any feedback people may have